src/SubNav.js

```diff
@@ -46,19 +46,19 @@ const SubNav = styled(SubNavBase)`
   ${sx};
 `
 
-SubNav.Links = Flex
+SubNav.Links = props => <Flex {...props} />
 
 SubNav.Link = styled.a.attrs(props => ({
   activeClassName: typeof props.to === 'string' ? 'selected' : '',
   className: classnames(ITEM_CLASS, props.selected && SELECTED_CLASS, props.className)
 }))`
   padding-left: ${get('space.3')};
   padding-right: ${get('space.3')};
-  min-height: 34px; // copied from TextInput, but should be a variable.
-  font-weight: ${get('fontWeights.bold')};
+  min-height: 34px;
+  font-weight: ${get('fontWeights.semibold')};
   font-size: ${get('fontSizes.1')};
-  line-height: ${get('lineHeights.default')};
-  color: ${get('colors.gray.6')};
+  line-height: 20px;
```

Comment:
SubNav.Link in Primer CSS uses a custom line height, so adjusted the line height here to match

Reply:
Should we leave comments in the code explaining why we've hard-coded the `min-height` and `line-height` values?

Reply:
done!